packages/cactus-plugin-htlc-eth-besu-erc20/src/main/typescript/web-services/refund-endpoint.ts

```diff
@@ -0,0 +1,86 @@
+import { Express, Request, Response } from "express";
+import {
+  Checks,
+  Logger,
+  LoggerProvider,
+  LogLevelDesc,
+} from "@hyperledger/cactus-common";
+import {
+  IExpressRequestHandler,
+  IWebServiceEndpoint,
+} from "@hyperledger/cactus-core-api";
+import { registerWebServiceEndpoint } from "@hyperledger/cactus-core";
+import {
+  PluginLedgerConnectorBesu,
+  Web3SigningCredentialType,
+  EthContractInvocationType,
+} from "@hyperledger/cactus-plugin-ledger-connector-besu";
+import HashTimeLockJSON from "../../solidity/contracts/HashedTimeLockContract.json";
+import { environment } from "../environment";
+
+export interface IRefundEndpointOptions {
+  logLevel?: LogLevelDesc;
+  connector: PluginLedgerConnectorBesu;
+}
+
+export class RefundEndpoint implements IWebServiceEndpoint {
+  public static readonly CLASS_NAME = "RefundEndpoint";
+  private readonly log: Logger;
+  private readonly connector: PluginLedgerConnectorBesu;
+
+  constructor(public readonly options: IRefundEndpointOptions) {
+    const fnTag = `${this.className}#constructor()`;
+    Checks.truthy(options, `${fnTag} arg options`);
+    const level = this.options.logLevel || "INFO";
+    const label = this.className;
+    this.log = LoggerProvider.getOrCreate({ level, label });
+    this.connector = this.options.connector;
+  }
+
+  public get className(): string {
+    return RefundEndpoint.CLASS_NAME;
+  }
+
+  public getVerbLowerCase(): string {
```

Comment:
@jordigiam Please make sure that if there's a typo in the verb name then the compiler bugs out with an error. (e.g. use the imported openapi JSON file)

Reply:
Done